packages/libs/eda/src/lib/workspace/DownloadTab/index.tsx

```diff
@@ -32,6 +32,8 @@ import { showLoginForm } from '@veupathdb/wdk-client/lib/Actions/UserSessionActi
 import { useHistory } from 'react-router';
 import { parsePath } from 'history';
 
+const EDA_USER_DATASET_PREFIX = 'EDAUD';
```

Comment:
Mind adding a comment here about where the prefix comes from? I'm guessing some serviec in the backend defines it? Or where should we look in case this changes?

Reply:
I've actually removed this code since the service is now sending us an `isUserStudy` attribute via https://github.com/VEuPathDB/EdaSubsettingService/pull/139